pkg/controllers/management/clusterstats/statsaggregator.go

```diff
@@ -198,6 +199,12 @@ func (s *StatsAggregator) aggregate(cluster *v3.Cluster, clusterName string) err
 	return nil
 }
 
+func minorVersion(cluster *v3.Cluster) (int, error) {
+	reg := regexp.MustCompile("[^0-9]")
```

Comment:
Since `MustCompile` could panic, it would be better to have this as a global variable.

Reply:
done